src/routes/votings/[id]/votes/+page.svelte

```diff
@@ -1,7 +1,112 @@
 <script lang="ts">
+	import { DefaultVoteOption, type CustomVoteOption } from '$models/voting.js';
+	import type {
+		CheckboxFilterGroup,
+		ComboboxFilterGroup,
+		SelectedCheckboxValueType
+	} from '$components/DataPage/DataPage.svelte';
+	import DataPage from '$components/DataPage/DataPage.svelte';
+	import VotingOptionTag from '$components/VotingOptionTag/VotingOptionTag.svelte';
+
 	export let data;
+
+	const { voting, filterOptions, votes } = data;
+
+	let comboboxFilterList: ComboboxFilterGroup[] = [
```

Comment:
Should be reactive `$`, because it depends on props.

Reply:
[45448](https://github.com/wevisdemo/parliament-watch/pull/71/commits/454480575787df503a93a7756c158d15f3d3e396)